### PR TITLE
GEODE-6229: Add ubuntu build image to Packer scripts

### PR DESCRIPTION
**Pull request by pdxcodemonkey**

### CODE REVIEW

File: cppcache/benchmark/CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set(THREADS_PREFER_PTHREAD_FLAG TRUE)
+
+find_package(Threads REQUIRED)
```
Comment (pivotal-jbarrett) [inline cppcache/benchmark/CMakeLists.txt]: Revert these changes since they are now correctly in dependencies/benchmark.

File: packer/ubuntu/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x -e -o pipefail
+
+apt-get autoclean
```
Comment (pivotal-jbarrett) [inline packer/ubuntu/cleanup.sh]: Add trailing lf.

File: sqliteimpl/CMakeLists.txt [hunk 3]
```diff
@@ -23,16 +23,20 @@ add_library(SqLiteImpl SHARED
   SqLiteImpl.hpp
 )
 
+if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```
Comment (pivotal-jbarrett) [inline sqliteimpl/CMakeLists.txt]: Why is this changing in this PR?

File: cppcache/benchmark/CMakeLists.txt (hunk 1 above)
Comment (pdxcodemonkey) [reply cppcache/benchmark/CMakeLists.txt]: Done

File: packer/ubuntu/cleanup.sh (hunk 2 above)
Comment (pdxcodemonkey) [reply packer/ubuntu/cleanup.sh]: fixed
